README.md

```diff
@@ -1,34 +1,19 @@
-<ul>
-  <li><a href="#marked">About</a></li>
-  <li><a href="#install">Installation</a></li>
-  <li><a href="#usage">Usage</a></li>
-  <li><a href="#specifications">Supported Markdown specifications</a></li>
-  <li><a href="#security">Security</a></li>
-  <li><a href="#contributing">Contributing</a></li>
-  <li><a href="#authors">Authors</a></li>
-  <li><a href="#license">License</a></li>
-</ul>
-
-<h2 id="marked">Marked</h2>
+# Marked
```

Comment:
I greatly simplified the main README and moved the more complex one to `/docs/README.md`